### PR TITLE
Ra 233

### PR DESCRIPTION
**Pull request by vguzmanp**

### CODE REVIEW

File: VAS.Services/ViewModel/EventTypeCollectionVM.cs [hunk 1]
```diff
@@ -37,5 +38,10 @@ public EventTypeCollectionVM ()
 			get;
 			set;
 		}
+
+		public AndPredicate<TEventVM> Filters {
+			get;
+			set;
+		} = new AndPredicate<TEventVM> ();
```
Comment (aameijide) [inline VAS.Services/ViewModel/EventTypeCollectionVM.cs]: Use constructor to initialize properties and document public properties

File: VAS.Services/ViewModel/EventTypeCollectionVM.cs [hunk 2]
```diff
@@ -16,6 +16,7 @@
 //  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 //
 using System;
+using VAS.Core.Filters;
 using VAS.Core.Interfaces.MVVMC;
 using VAS.Core.MVVMC;
 using VAS.Core.Store;
```
Comment (aameijide) [inline VAS.Services/ViewModel/EventTypeCollectionVM.cs]: unused using

File: VAS.Services/Controller/EventsController.cs [hunk 3]
```diff
@@ -85,6 +103,10 @@ public virtual IEnumerable<KeyAction> GetDefaultKeyActions ()
 
 		#endregion
 
+		protected virtual void HandlePlayerVMPropertyChanged (object sender, PropertyChangedEventArgs e)
+		{
+		}
```
Comment (aameijide) [inline VAS.Services/Controller/EventsController.cs]: Why we need this?

Comment (vguzmanp) [inline VAS.Services/Controller/EventsController.cs]: I have no idea, it's in the old one (https://github.com/fluendo/VAS/pull/127/files/c411c77a0f6219a09b3ddf545ddebfc106438969#diff-df4b6c0807fb84635787004f605d52f4L98)

**Pull request merged**